scripts/release-process/generate-changelogs.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const zip = require('lodash/zip');
+const { promisify } = require('util');
+const fs = require('fs');
+const path = require('path');
+const process = require('process');
+
+const template = require('./package-changelog-template.js');
+const { allChangelogPaths, readChangelog } = require('./util');
+
+const writeFileAsync = promisify(fs.writeFile);
+
+const determineOutputPath = changelogPath => {
+  const basename = path.basename(changelogPath, '.yml');
+  const markdownBasename = `${basename}.md`;
+
+  return path.join(path.dirname(changelogPath), markdownBasename);
+};
+
+const generateHumanReadableChangelog = parsedChangelog =>
+  template(parsedChangelog, 'bpk-component-spinner');
```

Comment:
This hard coded title `'bpk-component-spinner'` should be derived from the `changelog.yml` right?

Reply:
Good catch it should be derived from `package.json`. Let me fix that